docs/api.rst

```diff
@@ -518,7 +518,6 @@ general testing utilities.
     testing.EqualsTester
     testing.highlight_text_differences
     testing.nonoptimal_toffoli_circuit
-    testing.only_test_in_python3
```

Comment:
was this intentional?

Reply:
Building with this throws a `Not found` warning. Due to us removing Python 2, we deleted this procedure.